internal/utils/utils.go

```diff
@@ -44,9 +46,18 @@ var (
 		Steps:    5,
 	}
 
+	// Lightweight backoff for individual Prometheus queries (collector, etc.)
+	PrometheusQueryBackoff = wait.Backoff{
+		Duration: 500 * time.Millisecond,
+		Factor:   2.0,
+		Jitter:   0.1,
+		Steps:    4, // 500ms, 1s, 2s, 4s = ~7.5s total
```

Comment:
Minor comment here, comment can be updated to reflect actual backoff sequence with 4 steps